### DISCUSSION TITLE
[Question][Hw3] The job was not started because recent account payments have failed or your spending limit needs to be increased.

### DISCUSSION THREAD

**Discussion by Byunk** [prosyslab-classroom/cs348-information-security#696] [Q&A]
Name: Kyungho_Byoun

YOUR QUESTION
Github actions was failed with following message:
> The job was not started because recent account payments have failed or your spending limit needs to be increased. Please check the 'Billing & plans' section in your settings.

<img width="1233" alt="Screen Shot 2023-04-22 at 12 24 02 AM" src="https://user-images.githubusercontent.com/60650372/233674769-e04d44ec-6306-4310-8dec-e2c2b85981de.png">

Is it related to grading and submitting the assignment?

**Comment by yeonhee-ryou** [comment]
No. We provide GitHub Action just for your convinience and this is GitHub issue.
Grading will be done in Gradescope
